scripts/stac/imagery/capture_aera.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import Any, Dict, List
+
+from shapely import BufferCapStyle, BufferJoinStyle, Geometry, to_geojson, union_all
+from shapely.geometry import Polygon
+
+DECIMAL_DEGREES_1M = 0.00001
+"""
+Degree precision of ~1m (decimal places 5, https://en.wikipedia.org/wiki/Decimal_degrees)
+"""
+
+
+def to_feature(geometry: Geometry) -> Dict[str, Any]:
+    """Transform a Geometry to a GeoJSON feature.
+
+    Args:
+        geometry: a Geometry
+
+    Returns:
+        a GeoJSON document.
+    """
+    return {"geometry": json.loads(to_geojson(geometry)), "type": "Feature", "properties": {}}
```

Comment:
Do we need to specify the `properties` property?

Reply:
> A Feature object has a member with the name "properties".
https://datatracker.ietf.org/doc/html/rfc7946#section-3.2

The "has" seems a mandatory thing to me. But I could be wrong.

Reply:
> The value of the properties member is an object (any JSON object or a JSON null value).

Seems pretty clear it's mandatory if it can have a `null` value. :+1: